scst-store/multicluster-setup.hbs.md

```diff
@@ -64,24 +58,13 @@ data:
 EOF
 ```
 
-### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate from the View cluster
+### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate data from the View cluster
 
-With your kubectl targeted at the View cluster, you can view AMR CloudEvent Handler's TLS
-CA certificate. To copy the CA certificate into a `ceh_ca.yaml` file:
+With your kubectl targeted at the View cluster, you can get AMR CloudEvent Handler's TLS
+CA certificate's data.
```

Comment:
⛏️I feel the word data in this situation is extraneous which makes the phrase read a bit strange. I think think it's needed.

Reply:
There may be a few other places beyond these two that I pointed out if you plan on changing it.

Reply:
I know it feels odd but I kept it aligned with `ca_cert_data` in values file